git/clone/rwx-package.yml

```diff
@@ -154,6 +154,9 @@ tasks:
       CREDENTIAL_HELPER: ${{ tasks.setup.values.credential-helper }}
       FETCH_FULL_DEPTH: ${{ params.fetch-full-depth }}
       SUBMODULES: ${{ params.submodules }}
+      PATCHES_DIR: ${{ run.dir }}/.patches
+    filter:
+      ${{ run.dir }}: [.patches]
```

Comment:
I think the intention of the dynamic task is that we want to preserve the cache hit on the clone task that's doing the heavy lifting. That way if you're iterating against a static commit sha but with various patches, you'll cache hit the clone and then have a quick task to apply the patch. Passing the patch into this task will cause is to always cache miss when the patch changes though right? We have a way to bypass that, but it's internal / undocumented, so we should discuss.